modules/core/src/main/java/org/apache/ignite/internal/processors/odbc/SqlListenerUtils.java

```diff
@@ -310,4 +314,49 @@ public static String translateSqlWildcardsToRegex(String sqlPtrn) {
 
         return toRegex.substring(1);
     }
-}
\ No newline at end of file
+
+    /**
+     */
+    public static Timestamp convertWithTimeZone(Timestamp ts, TimeZone tzFrom, TimeZone tzTo) {
```

Comment:
Is it possible for `ts` to be null here?

Reply:
Fixed.